builder/builder.go

```diff
@@ -45,9 +45,9 @@ const (
 	// PhaseBuild is the phase text for the build phase.
 	PhaseBuild = "2. Build 🔧"
 	// PhasePush is the phase text for the push phase.
-	PhasePush = "3. Push ⏫"
+	PhasePush = "3. Push Summary ⏫"
 	// PhaseOutput is the phase text for the output phase.
-	PhaseOutput = "4. Local Output 🎁"
+	PhaseOutput = "4. Local Output Summary 🎁"
```

Comment:
Should we additionally remove the numbers? I introduced the numbers to indicate an order of operations - but that's no longer applicable since pushes and local outputs can take place any time now.

Reply:
that's a good point -- they no longer imply an order.

Reply:
her's an example with them all removed:

![image](https://user-images.githubusercontent.com/1806823/214373387-22477576-8e33-4a25-8a23-932b15230e15.png)